src/extensions/CoveoSearch/CoveoSavedSearch/CoveoSavedSearchFieldDisplay.js

```diff
@@ -45,7 +45,8 @@ function CoveoSavedSearchFieldDisplay({ sdk }) {
         return true;
       });
       const cq = get(data, "cq");
-      field.setValue({ query: pruned, cq });
+      const numberOfItems = get(data, "numberOfItems");
```

Comment:
Should have a default value if numberOfItems is undefined?

Reply:
Am defaulting the value in the other file.